Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -9,6 +9,38 @@
 
 namespace Hazel {
 
+	template <size_t N>
+	struct CleanExpressionResult
+	{
+		char Data[N];
+	};
+
+	template <size_t N>
+	constexpr auto CleanExpression(const char(&expr)[N])
```

Comment:
Rename to `RemoveCdecl`

```suggestion
	constexpr auto RemoveCdecl(const char(&expr)[N])
```

Reply:
Done.